README.md

```diff
@@ -91,6 +91,61 @@ sudo apt-get install protobuf-compiler libprotoc-dev
 pip install onnx
 ```
 
+### Windows
+When building on Windows it is highly recommended that you also build protobuf locally as a static library. The version distributed with conda-forge is a DLL and this is a conflict as ONNX expects it to be a static lib.
+
+#### Instructions to build protobuf and ONNX on windows
+Step 1 : Build protobuf locally
+```
+git clone https://github.com/protocolbuffers/protobuf.git
+cd protobuf
+git checkout 3.9.x
+cd cmake
+# Explicitly set -Dprotobuf_MSVC_STATIC_RUNTIME=OFF to make sure protobuf does not statically link to runtime library
+cmake -G "Visual Studio 15 2017 Win64" -Dprotobuf_MSVC_STATIC_RUNTIME=OFF -Dprotobuf_BUILD_TESTS=OFF -Dprotobuf_BUILD_EXAMPLES=OFF -DCMAKE_INSTALL_PREFIX=<protobuf_install_dir>
+msbuild protobuf.sln /m /p:Configuration=Release
+msbuild INSTALL.vcxproj /p:Configuration=Release
+```
+
+Step 2: Build ONNX
```

Comment:
Any instructions for a clean uninstall & rebuild? I know I had to do it some times, but I was using conda environments (not sure about outside).

Reply:
Indeed cmake doesn't support uninstall, neither Windows nor Linux.

Reply:
You may install the files into a separated folder and delete the whole folder afterwards.

Reply:
I believe there are some cleanup instructions for conda (removing the onnx folder from site-packages) which helps sometimes.

Reply:
For example, one mysterious problem I have run into a few times is the following: I add a new op, run setup. I then run the document-generator, but it seems to access the old schema registry, not the updated one with the new op. I never figured out what was going on.

Reply:
Yes there are a bunch of issues in this area too... We have a script update_docs... do you see the same problem when you run that script instead of independently running gen_doc? If yes then we should just fix that script to do the right thing... The existing doc for adding a new operator mentions this script.